apps/mudita-center-e2e/src/specs/overview/e2e-mock-sample.e2e.ts

```diff
@@ -74,6 +74,54 @@ describe("E2E mock sample - overview view", () => {
     await expect(drawerHeader).toBeDisplayed()
   })
 
+  it("Add third device", async () => {
```

Comment:
This test fails. I think you should not edit whole example, if you want you can create a new one.

🔴  
```
[0-0] Error in "E2E mock sample - overview view.Remove first device"
Error: element ("//p[contains(text(), 'Offline')]") still not displayed after 6000ms
```